src/effects/VST3/VST3Utils.cpp

```diff
@@ -10,14 +10,17 @@
 
 **********************************************************************/
 
+
 #include "VST3Utils.h"
+#include "Base64.h"
```

Comment:
Again you add an #include and should check whether you can remove the same file elsewhere.  And you can.

Reply:
done